### PR TITLE
History

### PR DESCRIPTION
**Pull request by SukanyaSen530**
Page
- History Page

Components - 
- Horizontal Card

Features -
- Video added to history
- Remove from history
- Clear All history

### CODE REVIEW

File: src/components/Cards/HorizontalCard/HorizontalCard.jsx [hunk 1]
```diff
@@ -1,5 +1,64 @@
...
+  const handleDelete = () => {
+    if (type === "history") {
+      removeFromHistory(_id, dataDispatch, openAlert);
+    }
+  };
+
+  return (
+    <article className="horizontal-card">
+      <Link to={`/explore/${_id}`} className="horizontal-card__img-container">
+        <img src={videoImage(_id, "lg")} className="img-responsive" />
```
Comment (ArjunGTX) [inline src/components/Cards/HorizontalCard/HorizontalCard.jsx]: An `alt` can be provided

**Review by sharathnair9999 — APPROVED** [review]
The code looks so readable. Good work.

**Review by ArjunGTX — APPROVED** [review]
Code Looks good. Good use of sass mixins

File: src/components/Cards/HorizontalCard/HorizontalCard.jsx [hunk 2]
```diff
@@ -1,5 +1,64 @@
...
+  const domNode = useClickOutside(() => setOpen(false));
+
+  const { dataDispatch } = useDataContext();
+  const {
+    globalHandlers: { openAlert },
+  } = useGlobalContext();
+
+  const handleDelete = () => {
+    if (type === "history") {
+      removeFromHistory(_id, dataDispatch, openAlert);
```
Comment (JowelTisso) [inline src/components/Cards/HorizontalCard/HorizontalCard.jsx:30]: You can extract "history" as constant like other constants you did

**Review by JowelTisso — APPROVED** [review]
* Clean code
* Looks Good

File: src/components/Cards/HorizontalCard/HorizontalCard.jsx (hunk 2 above)
Comment (SukanyaSen530) [reply src/components/Cards/HorizontalCard/HorizontalCard.jsx:30]: okay, will make it a constant in playlist PR.

File: src/components/Cards/HorizontalCard/HorizontalCard.jsx (hunk 1 above)
Comment (SukanyaSen530) [reply src/components/Cards/HorizontalCard/HorizontalCard.jsx]: fixed - 403a6375fe60a0e89b3f4ffa8ed6d1cbe82a16f6

**Pull request merged**